### ISSUE TITLE
Is this project still being worked on?

### ISSUE THREAD

**Issue by cyraid** [bazz1tv/snestracker#186]
I hate to be that guy and I don't mean any offense, but like the title says: Is this project still being worked on? I get that it may be just a hobby project, but I was under the impression there was patreon's and backers?

**Comment by bazz1tv** [comment]
@everyone Guys, I am officially putting down SNES Tracker. and if you would
like to talk about it with me, you can do so in my livestream right now, on
3 platforms - Twitch, Youtube, or Facebook.

https://twitch.tv/bazz1tv
https://youtube.com/bazz1tv
https://facebook.com/bazz1tv



My Discord is still a safe haven to hang out and share SNES Tracker related
things, music, SNES anything!

 And there is a chance someone else could further the project since it is
open source. ✨️

Yes i will be closing down the Patreon. Thanks everyone who supported me
along the way. I will be putting a long video post on the Patreon as my
tipping of the hat for everyone and last thoughts on the journey.

stay tuned for the video I will reply it here.

**Comment by aquamarine-axo** [comment]
Unless you've stopped working on it since April, the source is bieng kept private. If the latter is true, then anyone who wishes to continue this project will automatically be handicapped comparred to the paid prebuilt versions.
If, however, you're a looking for fully feature complete SNES tracker implementation, that isn't possible a GPL violation, Furnace Tracker has that.

**Comment by cyraid** [comment]
> Unless you've stopped working on it since April, the source is bieng kept private. If the latter is true, then anyone who wishes to continue this project will automatically be handicapped comparred to the paid prebuilt versions. If, however, you're a looking for fully feature complete SNES tracker implementation, that isn't possible a GPL violation, Furnace Tracker has that.

Does it support spc?

**Comment by aquamarine-axo** [comment]
> Does it support spc?

No, but that's coming for 0.7/8/9. It can load BRR (and save!) samples though, in addition to just loading from a wav.

**Comment by tildearrow** [comment]
> Guys, I am officially putting down SNES Tracker. and if you would like to talk about it with me, you can do so in my livestream right now, on 3 platforms - Twitch, Youtube, or Facebook.

I would strongly encourage you to release the source of the Patreon version before archiving this repo for the following reasons:
- preservation, and
- seeding a chance that the community may be able to continue developing it.

**Comment by livvy94** [comment]
I agree.

**Comment by vschwaberow** [comment]
It would be cool to integrate the Wiki sources into the project repository. Right now the link goes to a malicious web page.

**Comment by KungFuFurby** [comment]
From what I am aware of, the April source is the last known update on both the Pateron and the public end (v0.2.3 and v0.2.4 were uploaded at around the same time), with the version 0.3.0 WIP in its own branch.

I'll probably update the debugger myself, mostly because I use it more often than the actual tracker. However, I have both it and the modified libgme copy that is a submodule of this inside a `KFFFork` branch.
